tests/precalculations/test_text_metadata.py

```diff
@@ -8,57 +8,14 @@
 from shift_detector.precalculations.store import Store
 from shift_detector.precalculations.text_metadata import *
 
+import tests.utils.test_data as td
```

Comment:
It should be clear wether the utils package in tests contains test utils or tests for the utils. Right now it contains both

Reply:
done